src/DIRAC/Core/Utilities/ClassAd/ClassAdLight.py

```diff
@@ -61,7 +60,7 @@ def __analyse_jdl(self, jdl, index=0):
                 valuemode = 0
                 namemode = 1
 
-        return result
+        return S_OK(result)
```

Comment:
You would need to fix all the `return` to either return `S_OK` or `S_ERROR`. You are missing at least one.